src/tiled/addpropertydialog.cpp

```diff
@@ -46,24 +48,36 @@ AddPropertyDialog::AddPropertyDialog(QWidget *parent)
     mUi->setupUi(this);
     resize(Utils::dpiScaled(size()));
 
+
     // Add possible types from QVariant
     mUi->typeBox->addItem(typeToName(QVariant::Bool),    false);
     mUi->typeBox->addItem(typeToName(QVariant::Color),   QColor());
     mUi->typeBox->addItem(typeToName(QVariant::Double),  0.0);
     mUi->typeBox->addItem(typeToName(filePathTypeId()),  QVariant::fromValue(FilePath()));
     mUi->typeBox->addItem(typeToName(QVariant::Int),     0);
+    
     mUi->typeBox->addItem(typeToName(objectRefTypeId()), QVariant::fromValue(ObjectRef()));
     mUi->typeBox->addItem(typeToName(QVariant::String),  QString());
 
+
+    CustomProps customProps = Object::customProps();
```

Comment:
Should be `const CustomProps &customProps` to avoid reference counting and potential copying.